tests/regression/roles/sonic_bgp_neighbors/defaults/main.yml

```diff
@@ -564,6 +634,31 @@ merged_tests:
     state: merged
     input:
       - bgp_as: "{{bgp_as_1}}"
+        peer_group:
+          - name: SPINE
+            auth_pwd:
```

Comment:
Should a "modify" test case be added for the peer group "auth_pwd"?

Reply:
To modify, the auth_pwd has to be deleted first. The behavior is in the transformer code.

Reply:
Okay. Thanks for explaining. I will mark this conversation "resolved".